proto/responses.proto

```diff
@@ -188,8 +188,11 @@ message AccountProofsResponse {
     digest.Digest account_hash = 2;
     // Authentication path from the `account_root` of the block header to the account.
     merkle.MerklePath account_proof = 3;
+    // Account code, returned only in the case where the request code commitment does not match
+    // with the current one.
+    optional bytes account_code = 4;
     // State header for public accounts. Filled only if `include_headers` flag is set to `true`.
-    optional AccountStateHeader state_header = 4;
+    optional AccountStateHeader state_header = 5;
```

Comment:
I would probably put `account_code` inside `AccountStateHeader` (maybe we should rename it to something else).

The idea is that we'd include the code only if we also return the headers as getting just the code is probably not very useful.

Reply:
Changed, though couldn't think of a good name for this